tests/integration/helpers.py

```diff
@@ -109,11 +109,11 @@ async def push_to_unit(
     temp_filename_on_workload = _generate_random_filename()
     # unit does have scp_to
     await unit.scp_to(source=temp_path, destination=temp_filename_on_workload)  # type: ignore
-    mv_cmd = f"mv /home/ubuntu/{temp_filename_on_workload} {destination}"
+    mv_cmd = f"sudo mv -f /home/ubuntu/{temp_filename_on_workload} {destination}"
```

Comment:
Just out of curiosity what is the -f for?

Reply:
This is to force the file replacement without prompting to confirm